cobalt/browser/application.cc

```diff
@@ -223,6 +232,9 @@ GURL GetInitialURL(bool should_preload) {
     }
 #endif  // defined(ENABLE_ABOUT_SCHEME)
     GURL url = GURL(url_switch);
+    if (!url.is_valid()) {
```

Comment:
This will probably make the error weird if it's not a valid url, i.e. if I have a bad url `https://foo/bar` it would be logged that I have a bad url `https://https://foo/bar`—can we change it to only log the original url below?

Reply:
It's already logging only the `url_switch`, which is the unmodified value from the command-line.

I'll revisit this a part of the parsing investigation. Maybe there is a different way to allow URL without a scheme.